setup.py

```diff
@@ -53,173 +62,185 @@ class UploadCommand(BaseCommand):
 
     def run(self):
         self._run(
-            "Installing upload dependencies…",
+            "Installing upload dependencies ...",
             [sys.executable, "-m", "pip", "install", "wheel"],
         )
         try:
-            self.status("Removing previous builds…")
-            rmtree(os.path.join(here, "build"))
+            self.status("Removing previous builds ...")
             rmtree(os.path.join(here, "dist"))
+            rmtree(os.path.join(here, "build"))
         except OSError:
             pass
 
         self._run(
-            "Building Source and Wheel (universal) distribution…",
+            "Building Source and Wheel (universal) distribution ...",
             [sys.executable, "setup.py", "sdist", "bdist_wheel", "--universal"],
         )
-
         self._run(
-            "Installing Twine dependency…",
+            "Installing Twine dependency ...",
             [sys.executable, "-m", "pip", "install", "twine"],
         )
-
         self._run(
-            "Uploading the package to PyPI via Twine…",
+            "Uploading the package to PyPI via Twine ...",
             [sys.executable, "-m", "twine", "upload", "dist/*"],
         )
 
-        self._run("Creating git tags…", ["git", "tag", f"v{__version__}"])
-        self._run("Pushing git tags…", ["git", "push", "--tags"])
-
-
-class ValidateCommand(BaseCommand):
-    """Support setup.py validate."""
-
-    description = "Run Python static code analyzer (flake8), formatter (black) and unit tests (pytest)."
-
-    user_options = [
-        ('unit-test-target=', 'i', 'tests/{unit-test-target}'),
-        ('utt=', 'i', 'tests/{utt}'),
-        ('test-target=', 'i', 'tests/{test-target}')
-    ]
-
-    def initialize_options(self):
-        self.unit_test_target = ""
-        self.utt = ""
-        self.test_target = ""
 
+class CodegenCommand(BaseCommand):
     def run(self):
-        with open("./slack/web/client.py", "r") as original:
+        self._run(
+            "Installing required dependencies ...",
+            [sys.executable, "-m", "pip", "install"] + codegen_dependencies,
+        )
+
+        header = (
+            "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
+            "#\n"
+            "#  *** DO NOT EDIT THIS FILE ***\n"
+            "#\n"
+            "#  1) Modify slack_sdk/web/client.py\n"
+            "#  2) Run `python setup.py codegen`\n"
+            "#\n"
+            "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
```

Comment:
Who is this message/warning intended for?

Reply:
Maintainers/contributors may send patches modifying only `AsyncClient`.